lib/game.rb

```diff
@@ -0,0 +1,71 @@
+require_relative './modules/helper'
+require_relative './board'
+
+require 'set'
+
+class Game
+  extend Helper
+  include Helper
+
+  attr_reader :board
+
+  def initialize(player1, player2)
+    @player1 = player1
+    @player2 = player2
+    @board = Board.new
+    @choosed_nums = Set.new
+  end
+
+  private
+
+  def get_winner_name(combinations, board, player)
+    winner_name = player.name if combinations.all? do |num|
+      finded_element_from_board = board.find { |el| el.include? :"#{num}" }
+      finded_element_from_board[:"#{num}"] == player.sign
+    end
+    winner_name
+  end
+
+  public
+
+  def self.choose_signs
+    show_progress
```

Comment:
## Public and private [OPTIONAL]
- Add all the private method at the end of the class and the public at the beginning 
Example:

```ruby
class my_method
# not necessary to add 'public' statement 
# in here all public methods

private
 # in here all private methods

end
```